usl_models/usl_models/flood_ml/model.py

```diff
@@ -177,6 +178,27 @@ def train(
 
         return model_history
 
+    def load_model(self, filepath: str):
```

Comment:
can we add an annotation for the return type? Is it `-> 'FloodModel:'` ?

Reply:
Oops, the annotations didn't get pushed. This should be None.